x-pack/plugins/security/server/authorization/oss_features.test.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { KibanaFeature } from '../../../features/server';
+import { buildOSSFeatures } from '../../../features/server/oss_features';
+import type { LicenseType } from '../../../licensing/server';
+import { featurePrivilegeIterator } from './privileges';
+
+describe('OSS Features', () => {
+  const features = buildOSSFeatures({
+    savedObjectTypes: ['foo', 'bar'],
+    includeTimelion: true,
+    includeReporting: false,
+  });
+  features.forEach((featureConfig) => {
+    (['enterprise', 'basic'] as LicenseType[]).forEach((licenseType) => {
+      describe(`with a ${licenseType} license`, () => {
+        it(`returns the ${featureConfig.id} feature augmented with appropriate sub feature privileges`, () => {
+          const privileges = [];
+          for (const featurePrivilege of featurePrivilegeIterator(
```

Comment:
Is there a reason `featurePrivilegeIterator` cannot belong to `features` plugin?

Reply:
🤔 I can't think of a reason why it can't belong to the `features` plugin. It's only required by the `security` plugin today, but it's also not coupled to anything else within `security`. If you'd prefer to make this part of the `features` plugin contract, then I can move this out of `security`

Reply:
Yeah, let's move,  IMO it belongs to `features` plugin domain.